### PR TITLE
#4 implement sidebar in homescreen

### PR DESCRIPTION
**Pull request by Boti28**
helicopter go brrrrrrrrr with icons

### CODE REVIEW

File: client/lib/pages/SideBar.dart
Comment (Piratepie) [inline client/lib/pages/SideBar.dart:1]: change name of file to follow dart convention  
change to `sidebar.dart`

File: client/assets/deck1-logo.svg
Comment (Piratepie) [inline client/assets/deck1-logo.svg:1]: Make assets names consistent  
`lowerUperNoDash`

**Review by Piratepie — CHANGES_REQUESTED** [review]
Change the naming of certain files in `assets` and the `SideBar.dart` file to follow cameCase convention: `lowerUperNoDash`

**Review by ttommmy — APPROVED** [review]
Nothing else to add besides what @Piratepie  said.

**Review by Piratepie — APPROVED** [review]
nicuh

**Comment by tomassabol** [comment]
is text background implemented for the current page in navbar?

**Pull request merged**